pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1414,6 +1414,42 @@ self: super: {
   # Testsuite trying to run `which haskeline-examples-Test`
   haskeline_0_8_0_0 = dontCheck super.haskeline_0_8_0_0;
 
+  insert-ordered-containers = super.insert-ordered-containers.override {
+    optics-core = self.optics-core_0_3_0_1;
+    optics-extra = self.optics-extra_0_3;
+  };
+
+  # Tests depend on HTF and it is broken, 2020-08-23
+  list-t = dontCheck super.list-t;
+  superbuffer = dontCheck (doJailbreak super.superbuffer);
+  stm-containers = dontCheck super.stm-containers;
+
+  # Fails with "supports custom headers"
+  Spock-core = dontCheck super.Spock-core;
+
+  # Needed by Hasura  1.3.1
+  dependent-map_0_2_4_0 = super.dependent-map_0_2_4_0.override {
+    dependent-sum = self.dependent-sum_0_4;
+  };
+
+  # Hasura 1.3.1
+  graphql-engine = (super.graphql-engine.override {
+     immortal = self.immortal_0_2_2_1;
+     dependent-map = self.dependent-map_0_2_4_0;
+     dependent-sum = self.dependent-sum_0_4;
+     witherable = self.witherable_0_3_2;
+  }).overrideDerivation(d: rec {
+     # version in cabal file is invalid
+     version = "1.3.1-beta1";
```

Comment:
What makes you say that the version number would be invalid? If I'm not mistaken, then it's `1.0.0`, which looks like a perfectly valid version number.

Reply:
Their release versions are different, compared to what is specified in cabal file: https://github.com/hasura/graphql-engine/releases They are also using these versions as `VERSION` env variable during build, so they don't use version in cabal file at all. This gives only one conclusion, that they don't update version in cabal file, and yes consequently we need to patch version, regardless whether is wrong or not.

Reply:
OK, I see. Upstream is really jumping through all kinds of hoops here to make sure that people who distribute their software have a maximum hard time. :-(